mailpoet/lib/Automation/Integrations/MailPoet/Fields/NewsletterLinkFieldsFactory.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Automation\Integrations\MailPoet\Fields;
+
+use MailPoet\Automation\Engine\Data\Field;
+use MailPoet\Automation\Integrations\MailPoet\Payloads\NewsletterLinkPayload;
+
+class NewsletterLinkFieldsFactory {
+  public function getFields(): array {
+    return [
+      new Field(
+        'mailpoet:email_link:url',
```

Comment:
Should we use `email-link` instead of `email_link`? I'm not seeing any other fields that use underscores, and `email-link` would match `NewsletterLinkSubject::KEY`

Reply:
Thanks! Yeah, we should use -: https://github.com/mailpoet/mailpoet/pull/5255/commits/04d38f5d219e0f01ae64d9d429365b2248087471